rswag-specs/lib/rswag/specs/example_group_helpers.rb

```diff
@@ -52,10 +54,21 @@ def parameter(attributes)
       end
 
       def response(code, description, metadata = {}, &block)
-        metadata[:response] = { code: code, description: description }
+        metadata[:response] = { code: code, description: description, examples: {} }
         context(description, metadata, &block)
       end
 
+      def example_context(description, metadata = {}, &block)
+        context(description, metadata.merge(skip_formatter: true), &block)
+      end
+
+      def example_value(value)
```

Comment:
Wouldn't it be better to add `metadata[:response][:examples] ||= {}` here instead of [changing](https://github.com/rswag/rswag/pull/305/files#diff-7c43e7d1fb91b5c86d12f8c431b704ea535c00e16694677fda4ecb67b9277fc7R57)
 `#response`?

This would be consistent with [parameters](https://github.com/rswag/rswag/blob/8c4be0f247b0831c6cbcc5e8dc6f7ede684225e8/rswag-specs/lib/rswag/specs/example_group_helpers.rb#L48) and [headers](https://github.com/rswag/rswag/blob/8c4be0f247b0831c6cbcc5e8dc6f7ede684225e8/rswag-specs/lib/rswag/specs/example_group_helpers.rb#L77). Also, this way you wouldn't have to [change existing tests](https://github.com/rswag/rswag/pull/305/files#diff-940d36a104df7df89754f468e7b258ea77564d5abf68151eb0122b66f78297b9R114). And resulting Swagger documents won't have these empty `examples: {}` for every response without examples.